src/zabapgit_object_prog.prog.abap

```diff
@@ -13,6 +13,21 @@ CLASS lcl_object_prog DEFINITION INHERITING FROM lcl_objects_program FINAL.
     INTERFACES lif_object.
     ALIASES mo_files FOR lif_object~mo_files.
 
+  PRIVATE SECTION.
+    TYPES:  BEGIN OF ty_tpool_i18n,
+              language TYPE langu,
+              textpool TYPE textpool_table,
+            END OF ty_tpool_i18n,
+            tt_tpool_i18n TYPE STANDARD TABLE OF ty_tpool_i18n.
+
+    METHODS:
+    serialize_texts
+      IMPORTING io_xml TYPE REF TO lcl_xml_output
+      RAISING lcx_exception,
+    deserialize_texts
+      IMPORTING io_xml TYPE REF TO lcl_xml_input
+      RAISING lcx_exception.
```

Comment:
Hi @EduardoCopat,  use of method names with "i18n" is also an option. But then again existing names are also descriptive and quite precise. Besides, I've followed existing approach to naming convensions for (de)serializing for objects DOMA, DTEL.

P.S. One more reason is that not everyone might know what i18n means;) At least I didn't know about it a week ago.